00-Introducing-HOOMD-blue/03-Initializing-the-System-State.ipynb

```diff
@@ -0,0 +1,439 @@
+{
```

Comment:
<p>I would prefer <code>Np</code> or <code>N_particles</code>. You don't need to change this, but just so that it is clear what <code>N</code> represents. Alternatively, you could use,</p><pre class="ql-syntax" spellcheck="false">m = ...
N = 2 * m ** 3
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>done.</p>